scripts/decay.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import matplotlib.pyplot as plot
+
+def plotdecay(initial_amount, half_life, time):
+    """
+    Plots the decay of an isotope, if given the half-life, decay, and initial
+    number of atoms
+
+    Parameters:
+    initial_amount: initial number of atoms
+    half_life: half-life of isotope, in seconds
+    time: time isotope is decaying, in seconds
+
+    Returns:
+    none
+    """
+
+    time_steps = np.linspace(0,time, time)
```

Comment:
is `time,time` really clear for this function call? Maybe we want the number of timesteps to be a variable?

Reply:
That's a good point!  I'll add a timestep argument.